openapi/openapi.json

```diff
@@ -0,0 +1,116 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "Smart Kickers - OpenAPI 3.0",
+    "description": "Documentation describing API for Smart Kickers server.",
+    "contact": {
+      "url": "https://wiki.one.int.sap/wiki/display/saplabspoland/Beavers+Team"
```

Comment:
Idk if this is good way to share contact to you, because this link will work only for our employes.

Reply:
I added additional way of contact by email. Our main target will be SAP employees anyway so I think a link to our wiki page could stay in the documentation.